constants.js

```diff
@@ -1,5 +1,5 @@
-const { Enum } = require("./enum");
+import { Enum } from "./enum.js";
```

Comment:
Pardon my ignorance on this, but do we prefer to include the `.js` extension in `import` statements?

Reply:
Nope, you're right! I will remove it.

Reply:
Actually when I removed those, it broke on the front end I think because it was looking for `"text/plain"` files, not JS files.